dsp/benchmark/magnitude_u16o12_benchmark.c

```diff
@@ -0,0 +1,56 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+
+// The magic scaler is (32767.0 / 2047.0)
+#define MAGIC_SCALER 32.01514f
+#define CONVERT_AND_SCALE(__in) \
+({ \
+    uint16_t __out = abs(le16toh(__in) - 2048); \
+    ceil(__out * 32.01514f); \
+})
```

Comment:
These seem to be unused.

Reply:
oops, was going to use them there then decided not to.